### DISCUSSION TITLE
lotus sync is slow

### DISCUSSION THREAD

**Discussion by yzn5555** [filecoin-project/lotus#5651] [Build and Sync]
Server configuration is 8 core 64G ，Synchronization was normal before。Around February 17th, the synchronization became very slow after the node restarted automatically through the daemon process。
The node has not been synchronized to the time when the high-level daemon restarts the lotus program, causing the synchronization to start again。As a result, the distance from the official latest height is getting bigger and bigger。
Upgrade to the latest version still does not work
“systemctl status lotus-daemon”   In order to avoid memory overflow, limit to 55G

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
what kind of drives is the node running on?

**Comment by yzn5555** [comment]
Running on the ubuntu server, after checking, it is found that many peer-to-peer connections show that they are all IPs in China


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "filecoin-project/lotus"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2021年2月19日(星期五) 下午4:47
收件人:&nbsp;"filecoin-project/lotus"<lotus@noreply.github.com&gt;;
抄送:&nbsp;"1076813060"<1076813060@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [filecoin-project/lotus] lotus sync is slow (#5628)





 
what kind of drives is the node running on?

**Comment by hucs2542** [comment]
@yzn5555 我也正在研究filecoin,但是同步也是巨慢,我想问一下你用的是专线吗?还是普通的家用网络,我感觉现在能挖到矿的用的不单止是国内普通的宽带
